src/applications/personalization/profile-2/components/connected-apps/AdditionalInfoSections.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect } from 'react';
```

Comment:
import ordering nit: should put 3rd party stuff first (including lodash), then our own `@department-of-veteran-affairs` stuff, then `platform` or `applications` stuff, and finally relative imports